Update README.md (#289)
Esri/esri-leaflet-geocoder: README.md

## Terms and Conditions

In order use the ArcGIS geocoding service you need the following:

1. An [ArcGIS Developer account](https://developers.arcgis.com/sign-up/) or [ArcGIS Online account](https://www.esri.com/en-us/arcgis/products/arcgis-online/trial).
2. If your application uses an basemap layer service or other Esri data, you must display`Powered by`[`Esri`](http://esri.com) in the map attribution. See [Basemap attribution](https://developers.arcgis.com/documentation/mapping-apis-and-services/deployment/basemap-attribution/) for more details.

## Not-stored vs stored geocodes

> To determine if there is a cost associated with using the Esri Leaflet Geocoder and the ArcGIS geocoding service, if you have an ArcGIS Developer account, go to [Pricing](https://developers.arcgis.com/pricing/). If you have an ArcGIS Online account, go to [Credits by Capability](https://doc.arcgis.com/en/arcgis-online/administer/credits.htm#GUID-D309A4D0-43CD-4E58-A6DF-012A82A6D794).

Author: ```suggestion
To determine if there is a cost associated with using the Esri Leaflet Geocoder and the ArcGIS geocoding service, if you have an ArcGIS Developer account, go to [Pricing](https://developers.arcgis.com/pricing/), or, if you have an ArcGIS Online account, go to [Credits by Capability](https://doc.arcgis.com/en/arcgis-online/administer/credits.htm#GUID-D309A4D0-43CD-4E58-A6DF-012A82A6D794).
```

Author: @alaframboise All suggestions applied, but there was one grammatical issue with the if's, this may work better if we break it into a bulleted list for the 2 different options

Reviewer: I'll merge this change and we can switch it to bulleted list in a follow-up if necessary. Thanks!

In order to make use of the ArcGIS Online World Geocoding Service in your web application:

1. Please sign up for an [ArcGIS Developer account](https://developers.arcgis.com) or purchase an [ArcGIS Online Organizational Subscription](http://www.arcgis.com/features/plans/pricing.html).

> 2. Ensure that `Powered by`[`Esri`](http://esri.com) is displayed in the map attribution.

Reviewer 2: I'm confirming this with Robert Jensen.

> 2. Ensure that `Powered by`[`Esri`](http://esri.com) is displayed in the map attribution.

Reviewer 2: Confirmed this text with Robert.

## Cost

> ArcGIS geocode results have two different costs depending if the results you receive will be used only during your app session or stored permanently. Refer to the [ArcGIS Developer pricing page](https://developers.arcgis.com/pricing/) for more information regarding costs. See [Search for addresses and places](https://developers.arcgis.com/documentation/mapping-apis-and-services/search/) for more information on the available geocoding services.

Reviewer 2: ```suggestion
Some geocoding is free with the geocoding service, but it depends on the type of account you have and if you are not storing or storing geocoding results. To determine if there is cost, if you have an ArcGIS Developer account, go to [Pricing](https://developers.arcgis.com/pricing/). If you have an ArcGIS Online account, go to [Credits by Capability](https://doc.arcgis.com/en/arcgis-online/administer/credits.htm#GUID-D309A4D0-43CD-4E58-A6DF-012A82A6D794).
```

Author: need to fix the punctuation on this suggestion

> To store geocoding results, pass `forStorage: true` and a valid access token (see [Esri Leaflet Get Started](https://developers.arcgis.com/esri-leaflet/get-started/)).

Reviewer 2: ```suggestion
Note: To store geocoding results, pass `forStorage: true` and a valid access token (see [Esri Leaflet Get Started](https://developers.arcgis.com/esri-leaflet/get-started/)).
```
